### ISSUE TITLE
Running the command `source configure` gives the error: `configure:19: = not found`

### ISSUE THREAD

**Issue by susg** [nexB/scancode-toolkit#839]
OS: ubuntu 16.04 LTS
OS type: 64 bit
Python version: 2.7.12
Scancode version: 2.2.1
After cloning my fork locally and creating a new branch, when I ran the command: `source configure`, it gave the the error: `configure:19: = not found`.

**Comment by tdruez** [comment]
Same issue on macOS 10.12:
```
scancode-toolkit/ $ source ./configure
./configure:19: = not found
```

@susg a workaround is to run `configure` by itself and then `source` the virtualenv activate:
```
./configure
source bin/activate
```

**Comment by susg** [comment]
thanks @tdruez

**Comment by pombredanne** [comment]
@Chaitya62 a PR is welcomed!

**Comment by Chaitya62** [comment]
@pombredanne  On it.

**Comment by yash-nisar** [comment]
@pombredanne Can we close this since the issue has been resolved with ca561cf67182fe6144bf5652274e0cfd4fd4973e ?
@susg Can you confirm if your issue has been resolved ? If yes, can you please close this issue ?

**Comment by susg** [comment]
Hi @yash-nisar, I had this problem when I was using ubuntu 16.04 but now I am using macOS and  this problem did not occur when configuring it.

**Comment by pombredanne** [comment]
closed thanks to @Chaitya62 ' ca561cf
Thanks!
